server/tserver/src/main/java/org/apache/accumulo/tserver/compactions/CompactionService.java

```diff
@@ -132,8 +132,9 @@ public CompactionService(String serviceName, String plannerClass, Long maxRate,
 
     this.executors = Map.copyOf(tmpExecutors);
 
-    this.planningExecutor = ThreadPools.getServerThreadPools().getPoolBuilder("CompactionPlanner")
-        .numCoreThreads(1).numMaxThreads(1).withTimeOut(0L, MILLISECONDS).build();
+    this.planningExecutor =
+        ThreadPools.getServerThreadPools().getPoolBuilder("compaction.service.compaction.planner")
```

Comment:
No enum for this pool name?

Reply:
Found this pool and a couple others that I missed. 

Added constants for all of them